src/main/java/core/basesyntax/model/machine/Machine.java

```diff
@@ -1,9 +1,35 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
 public class Machine {
+    @Id
+    @GeneratedValue(
+            strategy = GenerationType.AUTO,
+            generator = "native"
```

Comment:
do we need the `generator` property?

Reply:
Yes, absolutely we have to have it, without this we have an additional table hibernate_siquence. 
https://vladmihalcea.com/why-should-not-use-the-auto-jpa-generationtype-with-mysql-and-hibernate/